clients/venice-push-job/src/main/java/com/linkedin/venice/spark/datawriter/jobs/DataWriterSparkJob.java

```diff
@@ -129,6 +134,7 @@ private Dataset<Row> getVsonDataFrame(SparkSession sparkSession, PushJobSetting
 
           final byte[] inputKeyBytes = recordReader.getKeyBytes(record._1, record._2);
           final byte[] inputValueBytes = recordReader.getValueBytes(record._1, record._2);
+          // final Long timestamp = recordReader.getRecordRMD(record._1, record._2);
```

Comment:
Is this expected? I think we can leave a note that it is ignored for Vson inputs and pass in a `-1`?